.github/workflows/jekyll-gh-pages.yml

```diff
@@ -41,6 +41,8 @@ jobs:
       - run: node common/scripts/install-run-rush.js install
       - run: npm run build --verbose
         timeout-minutes: 20
+      - name: doc generate
```

Comment:
Oh, I see...
So at the point that this action "runs" which should be triggered by the "checkin" (please correct me if I'm wrong) we not only "decorate" the pages here but actually "generate them -- nice!

Reply:
this seems to be the branches that could auto trigger the run, but didn't give the access of publish the doc